opm/simulators/flow_ebos_energy.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "config.h"
+
+#include <opm/simulators/flow_ebos_polymer.hpp>
```

Comment:
Here should be `flow_ebos_energy.hpp`, but any explanation that it did not cause problems?

Reply:
oops, mea culpa! the reason why it works fine is that file only contains free functions and these do not need to be forward declared.